### PR TITLE
JAMES-3065 remove remaining usages of Reactor types in public api

### PR DESCRIPTION
**Pull request by mbaechler**

### CODE REVIEW

**Comment by mbaechler** [comment]
@chibenwa thank you for your help

**Comment by chibenwa** [comment]
> @chibenwa thank you for your help

You're welcome. I love having fixups on my prs to ease the adoption, so that's always a pleasur to help ^^

File: server/protocols/webadmin/webadmin-core/src/main/java/org/apache/james/webadmin/routes/HealthCheckRoutes.java [hunk 1]
```diff
@@ -94,7 +96,7 @@ public void define(Service service) {
             message = "Internal server error - When one check has failed.")
     })
     public Object validateHealthChecks(Request request, Response response) {
-        ImmutableList<Result> results = executeHealthChecks();
+        List<Result> results = executeHealthChecks().collectList().block();
```
Comment (blackheaven) [inline server/protocols/webadmin/webadmin-core/src/main/java/org/apache/james/webadmin/routes/HealthCheckRoutes.java:99]: Not `ImmutableList`?

**Comment by chibenwa** [comment]
Merged

**Pull request closed without merging**